lib/readline.js

```diff
@@ -73,6 +79,12 @@ function Interface(input, output, completer, terminal) {
     throw new TypeError('Argument "historySize" must be a positive number');
   }
 
+  if (typeof crlfDelay !== 'number' ||
```

Comment:
Perhaps instead of throwing, this should just coerce the value into a number and apply a min/max bound if it's above a maximum threshold ... e.g.

```js
crlfDelay = Math.min(kMaxcrlfDelay, crlfDelay >>> 0);
```